File: Tricks/Normal/task01.cpp
/* Read in a double echo it, but with a minimum field width of 7 
 * and 3 digits after the decimal point. 
 * */
#include <iostream>
#include <iomanip>
#include <cstdio>

using namespace std;

int main(){
	double n;
	cin >> n;
	cout << setw(7) << setprecision(3) << fixed << n << endl;
	scanf("%lf", &n);
	printf("%*.*f\n",7,3,n);


}

File: Tricks/Normal/task01.java
/* Read in a double echo it, but with a minimum field
 * width of 7 and 3 digits after the decimal point.
 *
 * */
import java.util.*;

class Main {
	public static void  main(String[] args) {
		Scanner sc = new Scanner(System.in);
		double d = sc.nextDouble();
		System.out.printf("%7.3f\n", d);
	
	}
}
